### ISSUE TITLE
Does not display high resolution images

### ISSUE THREAD

**Issue by kerolos** [puzzledqs/BBox-Label-Tool#25]
I have images with those size 1944, 2592. which part of the code should I modify for displaying the whole image?
thanks in advance.

**Comment by juzhitao** [comment]
I have the same problem . do you resolve it ?

**Comment by leyuan** [comment]
Here is what I did for the code inside function `load_dir`, I need a larger container for my (832, 832) size image. 

```
        self.img = Image.open(imagepath)
        self.tkimg = ImageTk.PhotoImage(self.img)
        width = max(self.tkimg.width(), 832)
        height = max(self.tkimg.height(), 832)
        self.mainPanel.config(width = width, height = height)
        self.mainPanel.create_image((width/2, height/2), image = self.tkimg)
```

@kerolos @juzhitao

**Comment by toobaimt** [comment]
I added scrolling in my fork to get around the issue: [here](https://github.com/toobaimt/BBox-Label-Tool)
Hope this helps!
@kerolos @juzhitao

**Comment by holak0** [comment]
any solution please for this problem because I have the same ????

**Comment by toobaimt** [comment]
@holak0 try [this](https://github.com/toobaimt/BBox-Label-Tool)

**Comment by Shomitg** [comment]
@toobaimt this takes me to the same repo. Can you please point me to the forked repo that you mentioned with the aforementioned fix.

**Comment by toobaimt** [comment]
@Shomitg https://github.com/toobaimt/BBox-Label-Tool

**Comment by Shomitg** [comment]
Thanks @toobaimt but the issue still persists, not able to view the entire image of resolution 2346 × 186

**Comment by toobaimt** [comment]
@Shomitg the scrollbar doesn't help?

**Comment by toobaimt** [comment]
You might have to edit code a bit to accomodate bigger images

**Comment by Shomitg** [comment]
@toobaimt yeah I guess I'll have to edit the code. the scrollbar doesn't seem to be working.

**Comment by sakthigeek** [comment]
@toobaimt The scrollbar isn't working. @Shomitg Could you give the edited code to accomodate high res images?

**Comment by HaveAGitGat** [comment]
@toobaimt  Thanks!

**Comment by murtaza1514** [comment]
@toobaimt scroll bar is working but the next button isn't working. Can't go to the next image :p

**Comment by ArnaudNorel95** [comment]
Hey sorry but I don't understand where is the code which solve this problem (scroll down is enable for me) ? When I click the link you gave @toobaimt I just arrived to the original repository
